### PR TITLE
Feature/minecraft update

### PR DESCRIPTION
**Pull request by XianzheFan**
## 📑 Description

Run multi-agent dialogues on the aact server, saving a first-person perspective screenshot of the agent every 30 seconds. Each time the server sends dialogue content to the client ([https://github.com/XianzheFan/mindcraft-sotopia-multiagent](https://github.com/XianzheFan/mindcraft-sotopia-multiagent)), the client will send STATS, INVENTORY, and VISION_RESPONSE (by calling the OpenAI Vision API) back to the server.

## ✅ Checks
- [ ] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed
- [ ] Branch name follows `type/descript` (e.g. `feature/add-llm-agents`)
- [ ] Ready for code review

### CODE REVIEW

File: sotopia/database/serialization.py [hunk 1]
```diff
@@ -50,6 +50,9 @@ class AgentProfileWithPersonalInformation(BaseModel):
     mbti: str = Field()
     model_id: str = Field()
 
+    class Config:
```
Comment (XuhuiZhou) [inline sotopia/database/serialization.py:53]: why adding this line?

File: sotopia/database/persistent_profile.py [hunk 2]
```diff
@@ -36,11 +36,9 @@ class AgentProfile(JsonModel):
     secret: str = Field(default_factory=lambda: "")
     model_id: str = Field(default_factory=lambda: "")
     mbti: str = Field(default_factory=lambda: "")
-    tag: str = Field(
```
Comment (XuhuiZhou) [inline sotopia/database/persistent_profile.py:39]: why remove the tag?

File: sotopia/cli/install/install.py [hunk 3]
```diff
@@ -313,8 +314,11 @@ def install(
                     )
                     exit(0)
             else:
-                (Path(tmpdir) / "dump.rdb").rename(
-                    Path(directory) / "redis-data/dump.rdb"
+                # (Path(tmpdir) / "dump.rdb").rename(
+                #     Path(directory) / "redis-data/dump.rdb"
+                # )
+                shutil.move(
```
Comment (XuhuiZhou) [inline sotopia/cli/install/install.py:320]: Is it necessary to change this?

**Review by XuhuiZhou — CHANGES_REQUESTED** [review]
Can you rename the `examples/experimental/group_discussion` folder to something like `examples/experimental/mine_craft`? Just to avoid the overwriting the original examples

File: sotopia/database/serialization.py (hunk 1 above)
Comment (XianzheFan) [reply sotopia/database/serialization.py:53]: resolve UserWarning: Field "model_id"

File: sotopia/cli/install/install.py (hunk 3 above)
Comment (XianzheFan) [reply sotopia/cli/install/install.py:320]: `shutil.move()` can move files across different file systems. The commented-out code will throw an error on my computer.

File: sotopia/database/persistent_profile.py (hunk 2 above)
Comment (XianzheFan) [reply sotopia/database/persistent_profile.py:39]: Thank you, I'll add the tag back shortly.

Comment (XianzheFan) [reply sotopia/database/persistent_profile.py]: finished

**Comment by XianzheFan** [comment]
> Can you rename the `examples/experimental/group_discussion` folder to something like `examples/experimental/mine_craft`? Just to avoid the overwriting the original examples

sure.

File: sotopia/cli/install/install.py (hunk 3 above)
Comment (XuhuiZhou) [reply sotopia/cli/install/install.py:320]: cool, maybe remove the commented lines then

File: sotopia/database/persistent_profile.py [hunk 4]
```diff
@@ -79,11 +82,6 @@ class EnvironmentProfile(JsonModel):
     agent_constraint: list[list[str]] | None = Field(
         default_factory=lambda: None,
     )
-    tag: str = Field(
```
Comment (XuhuiZhou) [inline sotopia/database/persistent_profile.py:82]: same here don't remove the tag

**Review by XuhuiZhou — APPROVED** [review]

**Pull request merged**